common/brave_channel_info_linux.cc

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/common/brave_channel_info_linux.h"
+
+namespace brave {
+
+std::string GetChannelSuffixForDataDir() {
+  std::string modifier;
+  std::string data_dir_suffix;
+
+  char* env = getenv("CHROME_VERSION_EXTRA");
```

Comment:
do we have `brave-browser` PR that set this env?
It would be better if you link other depended PR in a PR. Just put depends `[url]` in commit or PR description.

Reply:
I wonder how can we trigger these targets
https://cs.chromium.org/chromium/src/chrome/installer/linux/BUILD.gn?l=517-525
so that we can have proper `CHANNEL` in` wrapper`?

Reply:
Sorry for late reply.
Afaik, we can do it by `yarn create_dist`.
With that, `//chrome/installer/linux` target is built.

Reply:
I mean how to control the channel setting, or is it build all available channels at once?

Reply:
Hmm, `create_dist` command doesn't have channel option.
Current `create_dist` would make all three channels at once.

Reply:
Should `create_dist` support channel option?

Reply:
I can accept current behavior and you can do channel option as follow-up

Reply:
Always thanks for valuable review! @darkdh